spec/text-editor-presenter-spec.coffee

```diff
@@ -20,6 +20,7 @@ describe "TextEditorPresenter", ->
 
       buffer = new TextBuffer(filePath: require.resolve('./fixtures/sample.js'))
       editor = atom.workspace.buildTextEditor({buffer})
+      editor.setUpdatedSynchronously(true)
```

Comment:
It wasn't needed, but it made the test failures deterministic so I could ensure it was fixed.